src/components/accessories/patientTherapy/patientTherapyTable/consts.ts

```diff
@@ -0,0 +1,15 @@
+export const header = [
+  "startDate",
+  "endDate",
+];
+export const label = {
+  startDate: "Start",
+  endDate: "End",
+  medicalId: "Medical",
+  qty: "Quantity",
+  freqInDay: "FID",
+  unitID: "unitID",
+  note: "Note",
+  freqInPeriod: "FIP",
```

Comment:
We should keep those strings inside a component in order to use the t() hook to get the Translations.

Reply:
It is done in another branch. It does'nt concern the delete feature.